### ISSUE TITLE
Bot trigger access

### ISSUE THREAD

**Issue by bsipocz** [bsipocz/bugbot#3]
The bot needs to be somewhat secure from being triggered by any snippets. The easiest to achieve this is to limit the trigger to maintainers. For this the open questions:

 - [x] the action only triggered when a label (e.g. bugbot) is present. As the labels can be only added by maintainers/people with triage access, it also ensures that the bug is reproducible with the given snippet.
 - [ ] other projects have bots that can be triggered by certain people by commenting on them. `baldrick` is not such a bot (yet), but it maybe worth looking at the either the `conda-forge-admin` or the `whedon` of joss to see how easy it is to list that functionality over to baldrick, or to any other reincarnation.
 
 - [ ] Is there github API access to see whether someone has write access (or team membership)

**Comment by ygrange** [comment]
https://developer.github.com/v3/teams/members/#list-team-members
Would that be the thing to look at for getting maintainer status?

**Comment by bsipocz** [comment]
Either having write access or looking for membership in a team (that is know to have write access)

**Comment by ygrange** [comment]
Good point on the teams. I saw the "role" thing and thought it may be the thing to look at for maintainers.

**Comment by bsipocz** [comment]
No, there isn't really such a role. Teams and individuals have certain level of access, and hopefully we will be able either to check that access level or membership in a certain team (that is within an organisation).
